dbt/adapters/snowflake/relation.py

```diff
@@ -120,3 +131,68 @@ def as_case_sensitive(self) -> "SnowflakeRelation":
                         path_part_map[path] = part.upper()
 
         return self.replace_path(**path_part_map)
+
+    def get_ddl_prefix_for_create(self, config: RelationConfig, temporary: bool):
```

Comment:
Nice, this looks like it will be easy to unit test

Reply:
I'd rather not shorten the names. Let's have them actually what they are doing. 

Also whoops, of course I can add return types!